src/Hamiltonians/HubbardMom1D.jl

```diff
@@ -1,32 +1,52 @@
 """
-    HubbardMom1D(address; u=1.0, t=1.0)
+    hubbard_dispersion(k)
+Dispersion relation for [`HubbardMom1D`](@ref). Returns `-2cos(k)`.
+
+See also [`continuum_dispersion`](@ref).
+"""
+hubbard_dispersion(k) = -2cos(k)
+
+"""
+    continuum_dispersion(k)
+Dispersion relation for [`HubbardMom1D`](@ref). Returns `2k`.
```

Comment:
Why it `returns '2k'` here? Wouldn't it be `returns 'k^2'`?

Reply:
Good catch.